vuln/npm/428.json

```diff
@@ -9,9 +9,9 @@
   "module_name": "base64url",
   "cves": [],
   "vulnerable_versions": "<=2.0.0",
-  "patched_versions": null,
-  "recommendation": "No fix is currently available for this vulnerability.\n\nIt is our recommendation to not install or use this module at this time.",
-  "references": "- https://hackerone.com/reports/321687",
+  "patched_versions": ">=3.0.0",
+  "recommendation": "Update `base64url` module to 3.0.0 or higher.",
```

Comment:
We're generally not wrapping the module name so just a plain text would be fine here